redistimeseries/client.py

```diff
@@ -42,11 +42,19 @@ def parse_m_range(response):
                                 parse_range(item[2])]})
     return res
 
+def parse_get(response):
+    if len(response) == 0:
+        return (None, None)
+    return (int(response[0]), float(response[1]))
+
 def parse_m_get(response):
     res = []
     for item in response:
-        res.append({ nativestr(item[0]) : [list_to_dict(item[1]),
-                                item[2][0], float(item[2][1])]})
+        if len(item[2]) is 2:
```

Comment:
Why "is" is used for comparison and not "==" ?
In this case, it will give the correct result, but it seems to me that using "is" is conceptually wrong here.
If I'm not mistaken, then python keeps an array of integer objects for all integers between -5 and 256.

```
l1 = list(range(2))
l2 = list(range(257))

print(len(l1) == 2) #True
print(len(l1) is 2) #True

print(len(l2) == 257) #True
print(len(l2) is 257) #False
```